### PR TITLE
Add remote ip to request log

### LINKED ISSUE #215: Add "X-Forwarded-For" header value to the log file.
Since there is no built in module to limit a number of login requests in verdaccio, it makes sense to use some log parser to find and block client with too many incorrect login attempts.

Unfortunately in the case of "behind the reverse proxy" setup, only proxy's IP address is logged.
Since the proxy is capable of sending X-Forwarded-For HTTP header it would be nice to have it included in the logs too.

### PR DESCRIPTION
**Pull request by Meeeeow**
**Type:** feature

The following has been addressed in the PR:

* There is a related issue

**Description:**
Add remote ip address to request log for security

Close #215

### CODE REVIEW

File: lib/middleware.js [hunk 1]
```diff
@@ -175,7 +175,10 @@ module.exports.log = function(req, res, next) {
   };
 
   const log = function() {
-    let message = '@{status}, user: @{user}, req: \'@{request.method} @{request.url}\'';
+    let forwardedFor = req.headers['x-forwarded-for'];
+    let remoteAddress = req.connection.remoteAddress;
+    let remoteIP = forwardedFor ? `${forwardedFor} via ${remoteAddress}` :  remoteAddress;
```
Comment (juanpicado) [inline lib/middleware.js]: I think you have some extra spaces in the last part of this expression

Comment (Meeeeow) [inline lib/middleware.js]: Yeah, you are right 👍 Fixed, Accidentally pressed keyboard before commit.

Comment (juanpicado) [inline lib/middleware.js]: Nah :) no problem,  i will merge soon and release 2.2.1.

Comment (Meeeeow) [inline lib/middleware.js]: Check your Gmail inbox :) I have send an email to you about new UI

**Pull request merged**